### ISSUE TITLE
Landmarks should have a unique role or role/label/title(i.e. accessible name) combination.

### ISSUE THREAD

**Issue by libmanuk** [uklibraries/UKL_HeaderFooter#88]
For Springshare pages, the search all guides has a role but not a unique label.  In terms of accessibility, it looks the same as the second search box that is for just the individual guide.

Similar problem with the footer.  Aria labels in footer sections given a role need to be unique, as they are currently the same.
